### PR TITLE
ft: #184361236 add daily check for expired products 🕐 🥫 

### PR DESCRIPTION
**Pull request by shyakadavis**
# Description

This P.R concludes the story #184361236, which is `The products should stop listing if the expiry date has passed`

It uses a cron job scheduled to run every day at 17:30 (5:30 PM) to check for expired products, update the status of those which are, and send relevant emails to the sellers of those products.

Fixes # (issue)

# Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or Enhancement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have created a loom video for the new feature or enhancement
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by emilenfc — APPROVED** [review]
LGTM

**Review by ISHIMWE-Aime — APPROVED** [review]
LGTM 
Keep doing great work @shyakadavis

**Review by mucyoalexaxel — APPROVED** [review]
Great Work @shyakadavis

**Review by SergeInGithub — APPROVED** [review]
**LGTM**

File: src/database/models/Product.model.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ const Product = sequelize.define<ProductInstance>(
       unique: false,
     },
     stock: {
-      type: DataTypes.ENUM("Available", "Out of Stock"),
+      type: DataTypes.ENUM("Available", "Out of Stock", "Expired"),
```
[the remark this answers is not in the document]
Comment (shyakadavis) [reply src/database/models/Product.model.ts:46]: Thanks for that. Had forgotten. 🦋

**Pull request merged**